plugins/filter_aws/aws.c

```diff
@@ -102,17 +101,34 @@ static void expose_aws_meta(struct flb_filter_aws *ctx)
                     "aws." FLB_FILTER_AWS_HOSTNAME_KEY,
                     ctx->hostname);
     }
+
+    for (i = 0; i < ctx->tags_count; i++) {
+        /* len("aws.ec2.tags.") = 13 */
+        env_tag_name_mem = flb_sds_create_size(13 + ctx->tag_keys_len[i]);
```

Comment:
I think I mentioned this earlier but a common pattern in this code base that I always try to use is:

```
descriptive_var_name = flb_sds_create_size...
if alloc failure...
tmp = flb_sds_printf...
if alloc failure...
descriptive_var_name = tmp
```

while not blocking, I think the code would be most readable if this pattern was used everywhere in this contribution.

Reply:
Apologies, fixed, https://github.com/fluent/fluent-bit/pull/6065/commits/c0dae76b8a61db99cefe06b42dfd6c0c67688518